v2/internal/frontend/desktop/windows/go-webview2/pkg/edge/create_env_native.go

```diff
@@ -12,7 +12,7 @@ import (
 	"golang.org/x/sys/windows"
 )
 
-func createCoreWebView2EnvironmentWithOptions(browserExecutableFolder, userDataFolder string, environmentCompletedHandle *iCoreWebView2CreateCoreWebView2EnvironmentCompletedHandler) error {
+func createCoreWebView2EnvironmentWithOptions(browserExecutableFolder, userDataFolder string, environmentCompletedHandle *iCoreWebView2CreateCoreWebView2EnvironmentCompletedHandler, additionalBrowserArgs string) error {
```

Comment:
Should we return an error or panic if `additionalBrowserArgs` is not empty and the old loader is used?

Reply:
I don't think so, it is not really an error, it's just ignored.

Maybe we should write a warning to the terminal, that the arguments are ignored with this loader?

Reply:
It's clearly something that a developer wanted but is ignored. 

Since we say we will drop support I would as a developer prefer to get an error if something doesn't work instead of just a silent ignore or just a warning.

Reply:
We might add more flags in the near future and those might change the behaviour in a more relevant way, e.g enabling some features. And then a silent ignore would just frustrate developers if it's silently ignored.

Reply:
Or let's just implement the support on the old loader by setting the env variable here `WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS` instead of using the EnvironmentOptions.

https://github.com/wailsapp/wails/blob/d3b4105f7598ba51501541ebd4ee564ca75a70c6/v2/internal/frontend/desktop/windows/go-webview2/webviewloader/native_module.go#L154-L164

Reply:
I didn't see that option. We couléd do that, but I don't know how we should do that.

Pass the addition arguments further down to the preventEnvAndRegistryOverrides?

Reply:
Yeah, just passing them should be fine.
You can also leave that away and I'll do it in a future PR.

Reply:
So for now leave it as is or should I throw an error?

Reply:
You can leave it as is and I'll add it.